src/ActiveLogin.Authentication.BankId.AspNetCore/Models/BankIdDefaultQrCodeGenerator.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.IO;
+using System.Net.Mime;
+using System.Reflection;
+using System.Text;
+using Microsoft.Extensions.Localization;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Models
```

Comment:
Does this really belong in the Models folder/namespace?

Reply:
I would like to move this and the interface to:
`ActiveLogin.Authentication.BankId.AspNetCore.Qr`

Reply:
Nice!